src/content/language/functions.md

```diff
@@ -266,6 +266,41 @@ assert(loudify('hello') == '!!! HELLO !!!');
 This example uses an anonymous function.
 More about those in the next section.
 
+## Function types
+
+We can specify the type of a function, which is known as a *function type*.
+It is actually the type of a first-class function object: An expression whose 
+type is a function type can only evaluate to a first-class function object.
+A function type is obtained from a function declaration header by replacing 
+the function name by the word `Function`. Moreover, we are allowed to omit 
+the names of positional parameters, but the names of named parameters cannot 
+be omitted. For example:
+
+<?code-excerpt "misc/lib/language_tour/functions.dart (function-type)"?>
+```dart
+void printElement(int element) => print(element);
+
+void doPrint(void Function(int) printFunction, {required List<int> list}) {
+  list.forEach(printFunction);
+}
+
+var list = [1, 2, 3];
+
+// Store `doPrint` in a variable and call it.
+void Function(void Function(int), {required List<int> list}) g = doPrint;
```

Comment:
Is it possible to use a simpler example here, rather than one which involves declaring the type of a function that receives a function as one of its parameters?

It's a good to mention that you can do this, don't get me wrong, but I'm wondering if there's a simpler example to start with? Maybe something out of the SDK, like Flutter's WidgetBuilder typedef?

Reply:
Perhaps the example could be adjusted to emphasize the simple function type `void Function(int)` which is used in line 284, as well as the more complex one in line 290?